.circleci/verbatim-sources/workflows-ecr-gc.yml

```diff
@@ -10,7 +10,7 @@
       - ecr_gc_job:
             name: ecr_gc_job_for_pytorch
             project: pytorch
-            tags_to_keep: "271,262,256,278,282,291,300,323,327,347,389,401,402,403,405,a8006f9a-272d-4478-b137-d121c6f05c83,6e7b11da-a919-49e5-b2ba-da66e3d4bb0a,f990c76a-a798-42bb-852f-5be5006f8026,e43973a9-9d5a-4138-9181-a08a0fc55e2f"
+            tags_to_keep: "271,262,256,278,282,291,300,323,327,347,389,401,402,403,405,a8006f9a-272d-4478-b137-d121c6f05c83,6e7b11da-a919-49e5-b2ba-da66e3d4bb0a,f990c76a-a798-42bb-852f-5be5006f8026,e43973a9-9d5a-4138-9181-a08a0fc55e2f,8fcf46ef-4a34-480b-a8ee-b0a30a4d3e59"
```

Comment:
Why do we need to keep `e43973a9-9d5a-4138-9181-a08a0fc55e2f`? For the sake of release branches?

Reply:
Yes we don’t want to remove the docker images for the release branch just yet

Reply:
Just to clarify: another reason that we don't want to remove previous tag because there might be PRs (which are not rebased on top of this) using those dockers?